docs/api.rst

```diff
@@ -38,4 +38,3 @@ Artifact
 
    optuna_dashboard.artifact.upload_artifact
    optuna_dashboard.artifact.file_system.FileSystemBackend
-   optuna_dashboard.artifact.boto3.Boto3Backend
```

Comment:
Does this line cause something issue?
https://optuna-dashboard.readthedocs.io/en/latest/_generated/optuna_dashboard.artifact.boto3.Boto3Backend.html

Reply:
Sorry, some issues occurred only on my env. I reverted it.